features/admin.connections.v1/pages/authenticator-edit.tsx

```diff
@@ -0,0 +1,371 @@
+                { connector?.description ? (
+                    <Popup
+                        content={ connector?.description?.replaceAll("{{productName}}", productName) }
+                        trigger={ <span>{ connector?.description?.replaceAll("{{productName}}", productName) }</span> }
+                    />
+                ) : (
+                    AuthenticatorMeta.getAuthenticatorDescription(connector?.id)
+                ) }
+                <DocumentationLink
+                    link={ getLink(`develop.connections.newConnection.
+                    ${connector?.name}.learnMore`) }
+                >
+                    { t("common:learnMore") }
+                </DocumentationLink>
+            </div>
+        );
+    };
+
+    return (
+        <ExtensionTemplatesProvider
+            shouldFetch={ hasApplicationTemplateViewPermissions }
+            resourceType={ ResourceTypes.APPLICATIONS }
+            categories={ ApplicationTemplateConstants.SUPPORTED_CATEGORIES_INFO }
+        >
+            <TabPageLayout
+                pageTitle="Edit Connection"
+                isLoading={ isConnectorDetailsFetchRequestLoading }
+                loadingStateOptions={ {
+                    count: 5,
+                    imageType: "square"
+                } }
+                title={ resolveConnectorName(connector) }
+                contentTopMargin={ true }
+                description={ resolveAuthenticatorDescription(connector) }
+                image={ resolveConnectorImage(connector) }
+                backButton={ {
+                    "data-componentid": `${_componentId}-page-back-button`,
+                    onClick: handleBackButtonClick,
+                    text: t("console:develop.pages.authenticationProviderTemplate.backButton")
+                } }
+                titleTextAlign="left"
+                bottomMargin={ false }
+                data-componentid={ `${_componentId}-page-layout` }
+            >
+                <EditConnection
+                    identityProvider={ connector }
+                    isLoading={ isConnectorDetailsFetchRequestLoading }
+                    onDelete={ handleIdentityProviderDelete }
+                    onUpdate={ handleAuthenticatorUpdate }
+                    data-componentid={ _componentId }
+                    template={ template }
+                    type={ templateId }
+                    isReadOnly={ isReadOnly }
+                    isAutomaticTabRedirectionEnabled={ isAutomaticTabRedirectionEnabled }
+                    setIsAutomaticTabRedirectionEnabled={ setIsAutomaticTabRedirectionEnabled }
+                    tabIdentifier={ tabIdentifier }
+                />
+            </TabPageLayout>
+        </ExtensionTemplatesProvider>
+    );
+};
+
+/**
+ * A default export was added to support React.lazy.
+ * TODO: Change this to a named export once react starts supporting named exports for code splitting.
```

Comment:
We are now using default exports and have no plans to go back to named exports. Therefore let's delete this comment.